### PR TITLE
removed oldoldstable-slim from testing matrix

### PR DESCRIPTION
**Pull request by benPearce1**
Also removed use of `set-output` in actions

### CODE REVIEW

File: BuildAssets/test-linux-package.sh [hunk 1]
```diff
@@ -15,6 +15,14 @@ if [[ "$OSRELID" == "rhel" && ( -z "$REDHAT_SUBSCRIPTION_USERNAME" || -z "$REDHA
   exit 1
 fi
 
+if [[ "$OSRELID" == "debian" ]]; then
+  OSRELVERSIONID="$(. /etc/os-release && echo $VERSION_ID)"
+  if [[ "$OSRELVERSIONID" == "9" ]]; then
+    # from https://unix.stackexchange.com/a/743865
+    echo "deb http://archive.debian.org/debian stretch main" > /etc/apt/sources.list
+  fi
+fi
```
Comment (benPearce1) [inline BuildAssets/test-linux-package.sh:24]: this was supposed to fix the apt-get update problem but it didn't seem to work, leaving it here for future reference

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -170,7 +170,7 @@ jobs:
         distro: [
           'debian:stable-slim',
           'debian:oldstable-slim',
-          'debian:oldoldstable-slim',
+          #'debian:oldoldstable-slim',
```
Comment (matt-richardson) [inline .github/workflows/build.yml:173]:
```suggestion
```

File: BuildAssets/test-linux-package.sh (hunk 1 above)
Comment (matt-richardson) [inline BuildAssets/test-linux-package.sh:24]: If it doesn't work, recommend deleting?
```suggestion
```

File: build/Build.cs [hunk 3]
```diff
@@ -124,7 +124,11 @@ class Build : NukeBuild
 
             File.WriteAllText(octoVersionText, fullSemVer);
             Console.WriteLine($"##[notice]Release version number: {fullSemVer}");
-            Console.WriteLine($"::set-output name=version::{fullSemVer}");
+            var outputFile = Environment.GetEnvironmentVariable("GITHUB_OUTPUT");
```
Comment (matt-richardson) [inline build/Build.cs:127]: OctoVersion should do this stuff automatically?

**Review by matt-richardson — COMMENTED** [review]
A few comments - I'll wait for your response before saying yay or nay.

File: build/Build.cs (hunk 3 above)
Comment (benPearce1) [reply build/Build.cs:127]: So we should drop the `set-output` line and have nothing in this build script?

Comment (matt-richardson) [reply build/Build.cs:127]: Honestly, dunno :)
Might need some investigation into how it works. I'm not much of a GHA expert.

Comment (benPearce1) [reply build/Build.cs:127]: this change means that the output is equivalent to what we had previously, without understanding what OctoVersion should or shouldn't be doing

Comment (matt-richardson) [reply build/Build.cs:127]: 👍 I'll leave it with you to decide what to do :)

**Review by matt-richardson — APPROVED** [review]
Nice one - 👍

**Pull request merged**